### DISCUSSION TITLE
Is it possible to search and show the columns from a specific Document Library?

### DISCUSSION THREAD

**Discussion by pmwhelan** [microsoft-search/pnp-modern-search#1358] [Q&A] [answered]
Hi
I would like to use PnP Modern Search v4 on one document library on my site.

I am unsure how to specify certain columns.

For example if my Document Library ("Docs") contains a Category and a Description how do I display those columns in the search results?

I have looked in Slots but I cannot see the Category and Description in there. Perhaps because it is unique to the Doc Lib?

Any help appreciated.

Thanks
P

**Comment by wobba** [comment, marked as answer]
You can start reading https://docs.microsoft.com/en-us/sharepoint/manage-search-schema and https://docs.microsoft.com/en-us/microsoft-365/community/how-do-site-columns-become-managed-properties-thus-available-for-search which explains a bit how columns become crawled properties which you can use in managed properties.
